proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -157,15 +158,43 @@ message MavenConfig {
   repeated RuntimeConfig additional_runtimes = 3;
 }
 
+message SwiftConfig {
+  // RuntimeLibrary describes the runtime library dependency of the generated code.
+  message RuntimeLibrary {
+    // The name of the runtime library package, github.com/apple/swift-protobuf.
+    string package = 1;
+    // The version of the runtime library, i.e. "1.21.0"
+    string version = 2;
+    // The products exported by the runtime library.
+    repeated string products = 3;
+    // The minimum compatible platform versions of the runtime library.
+    message Platforms {
+      // The minimum compatible version of the MacOS platform.
+      string macos = 1;
+      // The minimum compatible version of the iOS platform.
+      string ios = 2;
+      // The minimum compatible version of the tvOS platform.
+      string tvos = 3;
+      // The minimum compatible version of the watchOS platform.
+      string watchos = 4;
```

Comment:
where do we pull these values from? what happens when they're not set?

Reply:
We would need to specify them in the buf.plugin.yaml and source the information from any source plugin's own Package.swift manifest file. For example: https://github.com/bufbuild/connect-swift/blob/main/Package.swift#L21-L25